### ISSUE TITLE
Add individual permission check for `/settings` options

### ISSUE THREAD

**Issue by thomasmny** [thomasmny/BuildSystem#250]
### What problem does this feature solve?

Allows admins to block certain settings from being toggled

**Comment by nicokempe** [comment]
Do you mean something like this for the [SettingsInventory](https://github.com/thomasmny/BuildSystem/blob/master/buildsystem-core/src/main/java/de/eintosti/buildsystem/settings/SettingsInventory.java) at the InventoryClickEvent?

```java
case 13:
    if (!player.hasPermission("buildsystem.settings.disableinteract")) {
        player.sendMessage(ChatColor.RED + "You do not have permission to toggle this setting.");
        return;
    }
    settings.setDisableInteract(!settings.isDisableInteract());
    break;
```

**Comment by thomasmny** [comment]
yes, but I'd also like to display whether or not a player is even allowed to modify the setting
https://github.com/thomasmny/BuildSystem/blob/571ece79cc834d3efe19130a99e9a70e70c7a049/buildsystem-core/src/main/java/de/eintosti/buildsystem/world/modification/CreateInventory.java#L114

**Comment by thomasmny** [comment]
Implemented in #460
